README.md

```diff
@@ -24,4 +24,31 @@ poeditor: {
   }
 ```
 
+## Download all languages
+Another task is to download all languages, with minimum poeditor coverage.
+
+```
+poeditor: {
+  download_all: {
+    // Same configuration as download
+    download_all: {
+      dest: 'tests/?.json',
+
+      // Specify minimum coverage here
+      minimum_coverage: 40
+    }
+  },
```

Comment:
download_languages is not the best name
thats the purpose of poeditor
why not "languages" and nested is "all"